lms/djangoapps/philu_overrides/views.py

```diff
@@ -486,7 +487,7 @@ def login_user_custom(request, error=""):  # pylint: disable=too-many-statements
 
 @ensure_csrf_cookie
 @cache_if_anonymous('share_after_enroll',)
-def course_about(request, course_id):
+def course_about(request, course_id):  # pylint: disable=too-many-statements
```

Comment:
Rather than disabling warning, is it possible to split it two or more functions?

Reply:
That would increase the scope of this ticket in development as well as for QA.